### ISSUE TITLE
Permit suppression of the `aws configure sso` example command

### ISSUE THREAD

**Issue by richardkmichael** [aws/aws-cli#9132]
### Describe the feature

Concerning the example printed at the end of `aws configure sso`:

```
To use this profile, specify the profile name using --profile, as shown:

aws s3 ls --profile myProfile
```

Either:

- Add a flag to suppress it; or,
- Simply remove it, as it seems noisy to use a flag, and a suitably terse flag name is not obvious.

### Use Case

In our AWS configuration, `ListBuckets` is prohibited, so a user receives `AccessDenied` when running this command.

This confuses our staff (especially newcomers to the AWS CLI), who follow our setup instructions, run the suggested command (believing it to be a _test_), which then fails and they have the impression they made a mistake in the setup.

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### CLI version used

aws-cli/2.22.4 Python/3.12.7 Darwin/23.6.0 source/arm64

### Environment details (OS name and version, etc.)

macOS 14

**Comment by tim-finnigan** [comment]
Thanks for the feature request — I think the example command is helpful to show how users can reference the new profile. That being said, maybe there's a better example to use there than S3 — maybe [aws sts get-caller-identity](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/sts/get-caller-identity.html). Will leave this open for further tracking/discussion.

**Comment by richardkmichael** [comment]
@tim-finnigan I agree an example is helpful, and I considered suggesting a replacement but I couldn't think of one off-hand that would be almost certain to succeed.  Yours is a good one.
